src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -1242,8 +1248,31 @@ def remove_temp_test_dir(self):
             errors.append("Error removing temporary test files")
         return errors
 
+    def fail(self, msg=None):
```

Comment:
(pylint-arguments-renamed) Parameter 'message' has been renamed to 'msg' in overridden 'TestWithServers.fail' method